packages/terra-core-docs/src/terra-dev-site/doc/icon/AccessabilityGuide.2.doc.mdx

```diff
@@ -0,0 +1,110 @@
+import IconManufacturer from 'terra-icon/lib/icon/IconManufacturer';
+import IconAnnouncement from 'terra-icon/lib/icon/IconAnnouncement';
+import IconTile from 'terra-icon/lib/icon/IconTile';
+import IconList from 'terra-icon/lib/icon/IconList';
+import IconTreemap from 'terra-icon/lib/icon/IconTreemap';
+
+<Badge />
+
+# Accessibility Guide for Terra Icon
+
+<Notice variant="important" ariaLevel="2">
+
+Accessibility of icons is important because icons are ubiquitous. They can convey visual meaning or be used as a decorative or supportive element in a UI design and are an essential tool for creating intuitive user experiences that are appropriate for all languages. However, **if implemented improperly or without alternative content, they will not be understood by everyone**.
+
+</Notice>
+
+Since icons are images, they are not inherently accessible to all users, especially those with significantly impaired vision or blindness. Therefore, interface **icons must have alternative content**. Screen readers convey alternative content to users who have difficulty interpreting icons. Speech input devices also use alternative content to target specific icons. It is common for interfaces to use multiples of the same icon on a screen. Development and content teams must consider this use case to ensure all users can predictably understand the icon's purpose. **Users expect identical icons to perform the same function**. Similarly, screen reader users expect their software to identify identical icons consistently. Therefore, failure to provide consistent identification (visual or alternative content) can cause problems for all users. In addition, people who use only their voice to interact with technology also rely upon consistent alternative content behind the icons to ensure they can interact with controls.
+
+<Whitespace />
+
+## Accessibility Considerations:
+
+### Meaningful Icons
+
+#### Informative Icons
+
+Icons that convey information **must have alternative text in code** to ensure that assistive technologies pass the icon's meaning to the user.
+
+<InformativeIconExamples/>
+
+In this example the factory icon (<IconManufacturer/>) and announcement icon (<IconAnnouncement/>) are both informative. The factory icon has the meaning that _this text is an address_. The announcement icon has the meaning that _this text is an announcement_. However, neither icon conveys that meaning to users who cannot see them. Therefore, both icons code must include `title="<alternative text>"` to adequately convey their meaning to a screen reader user. The factory icon should have `title="Manufactured at"` and the announcement icon should have `title="Announcement"`. With these text alternatives added, screen reader users will hear _"&lt;Text Alternative&gt; &lt;Text&gt_":
+
+<p><em>"Manufactured at 2800 Rock Creek Pkwy Kansas City, MO 64117"</em></p>
+<p><em>"Announcement Facilities will closed on Monday"</em></p>
```

Comment:
Genuinely curious: Would this read to a screen reader user as 'the announcement facilities will be closed' or would it be understood as 'the text that follows the word "announcement" is the announcement'?

Reply:
What is read will slightly differ depending on the screen reader, but on macOS VoiceOver it will be closer to: _Image Announcement. Facilities will closed on Monday._